### ISSUE TITLE
HTTP Error 404: The specified blob does not exist.

### ISSUE THREAD

**Issue by pwn01** [inovachrono/Saavn-Downloader#45]
For some songs I get this error-
"Download Error : HTTP Error 404: The specified blob does not exist."

I think its related to cover/album art, beacuse those error songs were downloaded without cover art.

example- https://www.jiosaavn.com/song/sweater-weather/SDwCADBGTXQ

**Comment by monuyadav016** [comment]
Yes. I think it's related to missing cover art since the song downloads properly but without meta data.

**Comment by yogesh2022** [comment]
Here is the PR, it will fix most of the links, but some of the links are formed incorrectly 
https://github.com/monuyadav016/Saavn-Downloader/pull/50
